packages/apps/job-launcher/server/src/modules/webhook/webhook.service.ts

```diff
@@ -81,7 +84,7 @@ export class WebhookService {
 
     // Check if the request was successful.
     if (status !== HttpStatus.CREATED && status !== HttpStatus.OK) {
```

Comment:
`httpService.post` will throw `AxiosError` in case it's not `2xx` code, so it makes more sense to handle it like [this](https://github.com/humanprotocol/human-protocol/blob/8261d897ce2b9a4f0288a4ab68f547592cdf69b2/packages/apps/reputation-oracle/server/src/modules/webhook/webhook-outgoing.service.ts#L81-L95) and throwing just `Error`, because it's not http-facing service